src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/StorePal.Windows.Export.cs

```diff
@@ -102,6 +90,165 @@ public void MoveTo(X509Certificate2Collection collection)
             }
         }
 
+        public byte[] ExportPkcs12(Pkcs12ExportPbeParameters exportParameters, SafePasswordHandle password)
+        {
+            bool tryAes256sha256 = exportParameters is
+                Pkcs12ExportPbeParameters.Default or
+                Pkcs12ExportPbeParameters.Pbes2Aes256Sha256;
+
+            byte[] exported = ExportPkcs12(tryAes256sha256, password, out bool aes256Sha256);
+
+            // What we asked for and what we got are the same - so return it as-is.
+            if (tryAes256sha256 == aes256Sha256)
```

Comment:
If I'm reading this correctly: Pkcs12ExportPbeParameters.Default, on Windows, is AES-256/SHA-2-256; and it's not the same as cert.Export(X509ContentType.Pkcs12)... so the older method doesn't export with Default, but specifically 3DES/SHA-1?

If we decided somewhere that the existing methods would keep their algorithm but the new one would make Default be "newer"... OK.  Just making sure we decided that at some point.

Reply:
> If I'm reading this correctly: Pkcs12ExportPbeParameters.Default, on Windows, is AES-256/SHA-2-256

Right. `Default`, to me, made sense to start with the best. It seemed weird to say "Okay, the default is something that is outdated". Though we reserve the right to change what default means, realistically it's still a breaking change that would be difficult to justify.

Reply:
That interpretation of Default seems sound.  And it appeared to be consistent across the platforms (and tested as such).